invokeai/frontend/web/scripts/typegen.js

```diff
@@ -3,7 +3,7 @@ import fs from 'node:fs';
 
 import openapiTS from 'openapi-typescript';
 
-const OPENAPI_URL = 'http://127.0.0.1:9090/openapi.json';
+const OPENAPI_URL = 'http://192.168.5.199:9090/openapi.json';
```

Comment:
Is the change from the loopback interface to a hard-coded private IP address intentional?